### PR TITLE
Course features 3 up to 5

### PR DESCRIPTION
**Pull request by Ghassanooooo**
can you review the code, please!

### CODE REVIEW

File: views/mixins/courseForm.pug [hunk 1]
```diff
@@ -39,94 +39,41 @@ mixin courseForm
...
+        each featuresGroup, i in courseFormConfig.featuresConfig
+         .form-group
+          label.d-block(for=featuresGroup.fileField.for)=featuresGroup.fileField.label
+          if course && course.features[i] && course.features[i].icon
+            img.mw-150.mh-150(src="/images/"+course.features[i].icon)
+          input#exampleFormControlFile1.form-control-file(type=featuresGroup.fileField.type name=featuresGroup.fileField.name)
 
-          label(for='features_title_2') Features Title 2
-          input(type="text", name="features_title_2",  value=course ? course.features[1].title : '', placeholder="features title 2" )
+          label(for=featuresGroup.textFeildTitle.for)=featuresGroup.textFeildTitle.lable
```
Comment (spielhoelle) [inline views/mixins/courseForm.pug]: Please fix typo `textFeild` and `lable`

File: views/mixins/courseForm.pug [hunk 2]
```diff
@@ -39,94 +39,41 @@ mixin courseForm
...
-          input(type="text", name="features_subtitle_2",  value=course ? course.features[1].subtitle : '', placeholder="feature subtitle 2" )
-        .form-group
-          label.d-block(for='features3') Features icon 3
-          if course && course.features[2].icon
-            img.mw-150.mh-150(src="/images/"+course.features[2].icon)
-          input#exampleFormControlFile1.form-control-file(type='file' name='features_icon_3')
-          label(for='features_title_3') Features Title 3
-          
-          label(for='features_title_3') Features Subtitle 3
-          input(type="text", name="features_title_3",  value=course ? course.features[2].title : '' placeholder="features title 3")
```
Comment (spielhoelle) [inline views/mixins/courseForm.pug]: `textFeild` typo

**Comment by spielhoelle** [comment]
Good work! Just plz fix two typos + conflict from merging other branch, then I merge

File: views/mixins/courseForm.pug (hunk 1 above)
Comment (Ghassanooooo) [reply views/mixins/courseForm.pug]: I didn't got it 😅 can you refer to the typo

Comment (spielhoelle) [reply views/mixins/courseForm.pug]: `textFeildTitle` -> `textFieldTitle`
`lable` -> `label`

Comment (Ghassanooooo) [reply views/mixins/courseForm.pug]: Ops
thank you

Comment (spielhoelle) [reply views/mixins/courseForm.pug]: yw <3

**Pull request merged**